src/System.Private.CoreLib/src/System/Threading/LowLevelThread.cs

```diff
@@ -18,22 +19,16 @@ namespace System.Threading
 {
     internal static class LowLevelThread
```

Comment:
My bad, I read LowLevelLock instead of LowLevelThread (it's been a long week). I don't see any specific reason why this class needs to be there, I just left it there because it was there before. The functionality could be folded into other components. Wait-related stuff could go into WaitHandle.Windows.cs, and ReentrantWait-related stuff could be moved to RuntimeThread.